docs/paywall3.md

```diff
@@ -750,6 +751,17 @@ Here are listed and defined the Paywall 3.0 standalone functions:
 
 Table 4
 
+### `donations`
+
+This HTML class can be added to any HTML element. The `asset-id` represents the ID of a specific InPlayer asset. Usage example:
+
+```
+<button class="inplayer-paywall-donations" data-asset-id="{asset-id}" />
+```
+
+When clicking on this element, the donations flow for the provided asset is initiated by opening the donation options screen. 
+The donation options screen lists all donation options added to the asset that is being accessed. Upon selecting a donation option, with the user being logged in, the payment screen with the card payment method only available for donation will be shown. Once the payment is complete, a screen with a thank you message is displayed.
```

Comment:
I think the description is good, but too long. 

My suggestion:
... asset that is being accessed. After selecting a donation option the end-user can proceed with the donation payment.

What do you think?